package.json

```diff
@@ -31,16 +31,24 @@
   "homepage": "https://github.com/bamlab/android-performance-profiler#readme",
   "devDependencies": {
     "@types/jest": "^26.0.3",
+    "assert": "^2.0.0",
```

Comment:
seems like a lot of things got added here, which probably don't need to be? 🤔

Reply:
I'm not sure how those dependencies have been added,
I'll make sure to remove the unnecessary ones